src/is_face_detector/face_detection.py

```diff
@@ -0,0 +1,49 @@
+from is_wire.core import Logger
+from is_msgs.image_pb2 import Image, ObjectAnnotations
+import numpy as np
+import cv2
+
+
+
+class FaceDetector:
+    def __init__(self, op):
+        self.options = op
+        self.faceCascade = cv2.CascadeClassifier(op.model_file)
+        self.min_neighbors = op.cascade_options.min_neighbors
+        self.scale_factor = op.cascade_options.scale_factor
+
+    def detect(self, image):
+        # Read the image
+        self.image = image
+        gray = cv2.cvtColor(self.image, cv2.COLOR_BGR2GRAY)
+        self.faces = self.faceCascade.detectMultiScale(image=gray, 
+                                                       scaleFactor=self.scale_factor, 
+                                                       minNeighbors=self.min_neighbors)
+
+    def draw_detection(self, colors, line_width):
```

Comment:
put this function outside the class. This is a face detector class. 
Add the image as first parameter.
Put default values to colors and line-width.